app/controllers/users/unlocks_controller.rb

```diff
@@ -0,0 +1,28 @@
+class Users::UnlocksController < Devise::UnlocksController
```

Comment:
do we really need all these devise controllers filled with only comments - could we just delete the ones we're not actually customizing?